index.js

```diff
@@ -88,7 +93,7 @@ Datahub.prototype.getContent = function(channelUrl){
     throw new Error("Missing channel URL");
   }
 
-  return this._crud(channelUrl, 'GET');
+  return this._crud(this.config.url + channelUrl, 'GET');
```

Comment:
Perhaps I didn't name this function well enough, or perhaps there is a need for refactor here. The idea of the 'getContent' method was that I had a full callback URI that I was then passing to this method to get the content from that URI. There was no need for the URL in the config options since I had the full URI. And I thought that having some regex to parse out the channel name and query data was overkill. Ideas?